tests/test_permission_to_list_is_supported.py

```
@@ -0,0 +1,58 @@
+        birth_date=datetime.date(1985, 5, 5),
+        created_by=std_usr,
+    )
+
+    response = client.get("/api/gated-authors")
+    assert response.status_code == status.HTTP_200_OK, response.json()
+    data = response.json()
+    assert len(data) == 1
+    assert data[0]["name"] in ["Test Author 2"]
+
+
+# TODO(phuongfi91): What about listing permitted resources that have unpermitted related resources?
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)